richtextfx/src/main/java/org/fxmisc/richtext/TextExt.java

```diff
@@ -17,6 +17,11 @@
 import javafx.scene.shape.StrokeLineCap;
 import javafx.scene.text.Text;
 
+/**
+ * A class which adds some more styleable properties to JavaFX's Text class.
+ *
+ * They can be styled using the properties (and accessors/mutators) or via CSS.
+ */
```

Comment:
Moreover, I believe the added-on underline CSS properties will take effect even if `Text#isUnderline` is false. This should probably be stated at the class level and for each underline-related property